### ISSUE TITLE
[Feature Request]: Read network config from zerotier-one itself, not from database

### ISSUE THREAD

**Issue by BingMeng** [sinamics/ztnet#77]
### 🚀 Feature Summary

Read network config from zerotier-one itself

### 📝 Detailed Description

It is not good to store copies in database. Read from zerotier-one will make it stronger.

### 🎯 Use Case

Migrate from other controller -> lost all ui config.
Reinstall the zerotier-one -> config can't be sync.

### 💡 Willing to Contribute

None

**Comment by sinamics** [comment]
This has been on my todo list for a while. 
Fetching network should be from the controller and not DB. Will be improved in the next version.

**Comment by sinamics** [comment]
Just to elaborate on your request. 
As ztnet has support for multiple user accounts, each network needs to be in sync with DB / User. 
Thats why you will get issues when migrate from other controllers as those networks are not linked to your user. 

However, as an admin there should probably be a option for viewing all networks on the controller that is not linked to a user and manually add them.

**Comment by sinamics** [comment]
Added option to link un-managed network to a admin user. 
you can test the lastet main branch build if you want. `image: sinamics/ztnet:main-361b667`   

This build is only for amd64

**Comment by BingMeng** [comment]
> sinamics/ztnet:main-361b667

Thanks for your new build.
The new build has been tested, unassigned networks has been successfully added, members are showed correctly, but the options for members seem not synced (eg. Allow Ethernet Bridging).

**Comment by sinamics** [comment]
Thx
Next, i need to refine the backend fetching process to reduce its heavy reliance on the database. The database should only contain fundamental network and member details, with the primary fetching being executed by the controller itself.

**Comment by sinamics** [comment]
New version on its way. 0.3.1
https://github.com/sinamics/ztnet/releases/tag/v0.3.1

Data from controller has now higher priority. Fewer parameters is stored in db.
When migrating from a custom zerotier server, all the memer option will be from the controller. 

However, a current limitation persists: Migrating networks with previously attached FlowRules remains challenging. The local ZeroTier controller API doesn't offer a method to save `RulesSource`, making retrieval unfeasible.
In other words, Tags, Caps must be set from scratch.

I hope this will close this issue, if not, let me know.
